### ISSUE TITLE
Inconsistency between release.values and environment.values

### ISSUE THREAD

**Issue by mattparkes** [roboll/helmfile#413]
I'm writing a medium-complexity helmfile and the inconsistency between the `values` key in a `release` and the `values` key in an `environment` caught me by surprise.

 `README.md` shows that `releases.values` can contain a mixture of filepaths and key:value pairs:
```releases:
     - name: foo
       values:
      - vault.yaml
      - address: https://vault.example.com
        db:
          username: {{ requiredEnv "DB_USERNAME" }}
```
I tried using the same syntax in my `environments` block:
```
environments:
  staging:
    values:
      logging: "elasticsearch"
```

or

```
environments:
  staging:
    values:
      - logging: "elasticsearch"
```

and got an error like:

```
could not deduce `environment:` block, configuring only .Environment.Name. error: failed to read helmfile.yaml: reading document at index 1: yaml: unmarshal errors:
  line 4: cannot unmarshal !!map into []string
```

I know I can move the values to a yaml file to solve the issue, but it would be nice if this was consistent.

**Comment by davidovich** [comment]
I believe this is a duplicate of #359. 

It would be useful I agree, but it is not implemented yet.

**Comment by mumoshu** [comment]
@mattparkes @davidovich #359 has been finally implemented via #621. I'm closing this as resolved, but please feel free to reopen if it doesn't work for you!
